doc/tutorial/climada_entity_Exposures_osm.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to get exposure data from OpenStreetMap\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "[OpenStreetMap data](https://www.openstreetmap.org/) is a freely-accessible and valuable data source which can provide information on the geolocations of a variety of assets such as critical infrastructures, buildings, or ecosystems. Such data can then be used within the risk modelling chain of CLIMADA as exposures. In this tutorial we show how to retrieve exposure data by querying the [OpenStreetMap database](https://www.openstreetmap.org/) using the [osm-flex](https://osm-flex.readthedocs.io/en/latest/?badge=latest) python package."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quick example\n",
+    "Here we provide a quick example of an impact calculation with CLIMADA and OSM data.\n",
+    "We use in this example main roads in Honduras as exposures, and winter storms\n",
+    "as hazard. We load the osm data using `osm-flex` and disaggregate the exposures, \n",
+    "compute the damages, and reaggregate the exposures to their original shape using the \n",
+    "function `calc_geom_impact` from the util module `lines_polys_handler`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/lseverino/miniforge3/envs/climada_osm/lib/python3.9/site-packages/dask/dataframe/_pyarrow_compat.py:15: FutureWarning: Minimal version of pyarrow will soon be increased to 14.0.1. You are using 12.0.1. Please consider upgrading.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import osm_flex\n",
+    "import osm_flex.download\n",
+    "import osm_flex.extract\n",
+    "\n",
+    "from climada import CONFIG\n",
+    "\n",
+    "osm_flex.enable_logs()\n",
+    "\n",
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "import logging\n",
+    "from climada.util.config import LOGGER\n",
+    "\n",
+    "LOGGER.setLevel(logging.ERROR)"
```

Comment:
Add a comment in the code to explain that this is not needed and just used here for the nice visualization of the tutorials. People should in general not silence warnings unless they mean to.